security/remote_devices/ssh-tunneling/README.md

```diff
@@ -0,0 +1,7 @@
+# Reference Implementaton for SSH Tunneling on EdgeX Device Services
+
+This is a reference implementation for using SSH Tunneling on EdgeX device-virtual services.
+
+## Build and Run
+
+Please see the detailed steps in "how-to guide" for SSH Tunneling in documentation repository here: <https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/microservices/security/Ch-SSH-Tunneling-HowToSecureDeviceServices.md#put-it-all-together>.
```

Comment:
Note: this is the future hyperlink after merging into master of that repo.  Currently please use [this URL for reference](https://github.com/jim-wang-intel/edgex-docs/blob/device-ssh-tunneling-how-to/docs_src/microservices/security/Ch-SSH-Tunneling-HowToSecureDeviceServices.md#put-it-all-together)